scrapers/Xevunleashed.yml

```diff
@@ -0,0 +1,34 @@
+name: "XevUnleashed"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - xevunleashed.com/updates/
+    scraper: sceneScraper
+
+
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //span[@class="update_title"]/text()
+      Date:
+        selector: //span[@class="availdate"]/text()
+        concat: " "
+        postProcess:
+          - replace:
+              - regex: (\d{2}\/\d{2}\/\d{4}).*
+                with: $1
+          - parseDate: 01/02/2006
```

Comment:
```yaml
        selector: //span[@class="availdate"]/text()[1]
        postProcess:
          - parseDate: 01/02/2006
                 selector: //span[@class="availdate"]/text()[1]
        postProcess:
          - parseDate: 01/02/2006
  ```
Should be enough
The first text is always the date part

Reply:
Nice, didn't know that worked, will update asap